internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -38,8 +41,12 @@
 {{- end}}
 {{- end}} {{/* end if eq .WorkloadType "Load Balanced Web Service"*/}}
 {{- if eq .WorkloadType "Backend Service"}}
+{{- if .ServiceConnect }} # remove when release
+  PortMappings: !If [ExposePort, [{ContainerPort: !Ref ContainerPort, Name: !Ref WorkloadName}], !Ref "AWS::NoValue"]
```

Comment:
hmm I wonder if this works well with people that has task def overrides in their manifest

Reply:
Possible if they override the whole portmapping but should be fine. it's not something we can guarantee.